Bio/GenBank/__init__.py

```diff
@@ -1051,13 +1051,31 @@ def location(self, content):
                 cur_feature.location = SeqFeature.FeatureLocation(int(s) - 1,
                                                                   int(e),
                                                                   strand)
-            except ValueError as e:
+            except ValueError:
```

Comment:
Good change - I can see why single letter variables names are frowned on, here ``e`` is for ``error`` was it was also used just above for ``end``.